Change card.html to unescape html characters

earlng opened the pull request:
In the posts cards (the summary of blog posts) `plainify` doesn't properly escape the HTML so characters like double-quotes ("") are displayed as `&ldquo;` [image](https://imgur.com/jUTtXwk).

Changing to `htmlUnescape` resolves this problem

earlng committed:
Update card.html

removing "plainify" because it causes the quotes symbols (i.e ", ") to be displayed as the HTML entity name ldquo or ldquo

earlng committed:
Update card.html

use `htmlUnescape` instead of leaving it blank, as suggested https://discourse.gohugo.io/t/how-to-remove-html-tags-in-lead-without-using-plainify/27733

dholbach reviewed (approved):
This fix works great!

earlng committed:
card featured image should crop instead of resize

Use .Resize will make the featured images seem distorted when displayed on cards if the featured images do not match the aspect ration of 700x350. Using .Fill instead will crop the image so that it does not look distorted.

Source: https://gohugo.io/content-management/image-processing/#smart-cropping-of-images

earlng commented:
I'm sorry for the constant changes. I'm just gradually finding some minor improvements as I'm using your theme on my own website.

puresyntax71 commented:
Thanks! I guess this affects #102 as well.
